Hearthstone Deck Tracker/LogReader/Handlers/PowerHandler.cs

```diff
@@ -309,7 +309,7 @@ public void Handle(string logLine, IHsGameState gameState, IGame game)
 								AddKnownCardId(gameState, NonCollectible.Neutral.SparkDrill_SparkToken, 2);
 								break;
 							//TODO: Augmented Elekk
-							case Collectible.Neutral.HakkarTheSoulflayer:
```

Comment:
Shouldn't it be the same case for both Hakkar itself and the Corrupted Blood Token?

Reply:
The triggers are different - the first shuffle is a standard Deathrattle trigger with a known card, which doesn't need any special handling. It's just the Cast when drawn part that needs special care here.

Reply:
Yes, posting a gist with the power.log of a full game would be very good. Thanks.

Reply:
@riQQ Sorry for the late reply. See https://gist.github.com/Danstahr/0303e5f0f66ef84c520e11fbfa0d6f67. There's a few bookmarks in the log, search for `===`.